### ISSUE TITLE
Footnotes do not render as numbers, but as bullet points

### ISSUE THREAD

**Issue by jjhartman** [mtn/cocoa-eh-hugo-theme#10]
I've noticed that footnotes written in markdown, e.g. [^1] neither display as superscripts in the body texts, nor do the companion notes display as numbers.  Instead, they are all bullet points.  Furthermore, clicking on a body footnote will not redirect to the note at the bottom of the page, but will return to the homescreen.  Looking at the HTML and CSS files that Hugo generates, I can't seem to find any indication of why this happens, although I have compared it to another Hugo theme that doesn't have this issue.

Here's the HTML used in Cocoa Enhanced:
```

<div class="footnotes">

<hr />

<ol>
<li id="fn:1">FOOTNOTE TEXT 
 <a class="footnote-return" href="#fnref:1"><sup>[return]</sup></a></li>
```

The HTML matches that generated by other themes that render the footnotes as numbers, and looks fine, but perhaps there is something wrong with the definitions in the CSS file, the js files, or something else?  Screenshot attached.
![screenshot 28](https://cloud.githubusercontent.com/assets/25942525/23200457/71d305c8-f8a2-11e6-8e22-6ae5828d8912.png)

**Comment by fuegoio** [comment]
Hi ! 
Thanks for your issue, I will investigate it !

**Comment by fuegoio** [comment]
So it was a bit of CSS yes, and a bit of HTML configuration for the anchors links.

The `<base>` tag was bringing a bit of mess, so I removed it and adapted links with absolute paths in order to make it works. Images have to be absolute too now so be careful !

Fixed in b89ca391a6c290bb2f3c75c906d5c5b379e55296
